core/jobs/batch_jobs/exp_validation_jobs_test.py

```diff
@@ -0,0 +1,957 @@
+            param_changes=[],
+            auto_tts_enabled=feconf.DEFAULT_AUTO_TTS_ENABLED,
+            correctness_feedback_enabled=False,
+            states={"EXP_3_STATE_1": self.EXP_3_STATE_1}
+        )
+
+    def test_run_with_no_models(self) -> None:
+        self.assert_job_output_is([])
+
+    def test_run_with_state_validation(self) -> None:
+        self.put_multi([self.exp_1])
+        self.assert_job_output_is([
+            job_run_result.JobRunResult.as_stdout('EXPS SUCCESS: 1'),
+            job_run_result.JobRunResult.as_stderr(
+              "The id of exp is 1, and the state RTE erroneous data are " +
+              "[{'state_name': 'EXP_1_STATE_1'}, {'state_name': " +
+              "'EXP_1_STATE_2'}]"
+            ),
+            job_run_result.JobRunResult.as_stderr(
+              "The id of exp is 1, and the state interaction erroneous data " +
+              "are [{'state_name': 'EXP_1_STATE_1'}, {'state_name': " +
+              "'EXP_1_STATE_2'}]"),
+            job_run_result.JobRunResult.as_stderr(
+              "The id of exp is 1, and the state erroneous data are " +
+              "[{'state_name': 'EXP_1_STATE_1', " +
+              "'tagged_skill_misconception_ids': ['The " +
+              "tagged_skill_misconception_id of answer group 0 is not None.']" +
+              ", 'wrong_labelled_as_correct_values': ['The value of " +
+              "labelled_as_correct of answer group 0 is True but the " +
+              "destination is the state itself.'], 'not_sinle_rule_spec': " +
+              "['There is no rule presentin answer group 0, atleast one " +
+              "is required.'], 'invalid_destinations': ['The destination " +
+              "Not valid state of answer group 0 is not valid.'], " +
+              "'invalid_default_outcome_dest': ['The destination of default " +
+              "outcome is not valid, the value is Not valid state']}, " +
+              "{'state_name': 'EXP_1_STATE_2', " +
+              "'tagged_skill_misconception_ids': ['The " +
+              "tagged_skill_misconception_id of answer group 0 is not None.'],"
+              +" 'wrong_labelled_as_correct_values': ['The value of " +
+              "labelled_as_correct of answer group 0 is True but the " +
+              "destination is the state itself.'], 'not_sinle_rule_spec': " +
+              "['There is no rule presentin answer group 0, atleast " +
+              "one is required.'], 'invalid_destinations': ['The destination "+
+              "Not valid state of answer group 0 is not valid.'], " +
+              "'invalid_default_outcome_dest': ['The destination of " +
+              "default outcome is not valid, the value is Not valid state']}]"
+            )
+        ])
+
+    # def test_run_with_state_interaction_validaton(self) -> None:
+    #     self.put_multi([self.exp_2])
+    #     self.assert_job_output_is([
+    #         job_run_result.JobRunResult.as_stdout('EXPS SUCCESS: 1')
+    #     ])
+
```

Comment:
I have commented this section as this is the one causing the problem, for the context this particular test is to check for the invalid state interactions. I have created several states in the 2nd exploration. I added previously the `job_run_result.JobRunResult.as_stderr()` part to check what validation errors will occur and to check if I have created the states in the right way but I got this error `ValueError: JobRunResult must not exceed 1500 bytes` so I tried splitting all the validations as previously I was outputting them into a single `JobRunResult` then again I got the same error so I tried to check only for single validation in the interaction part and commenting out the rest of the checks but then also I got the same error.
I'm a bit confused about why this is happening @vojtechjelinek
Thanks